docs/reference/commandline/build.md

```diff
@@ -457,3 +459,115 @@ space.
 storing two copies of the image, one for the build cache with all the cache
 layers in tact, and one for the squashed version.
 
+#### Pre-Requisites
+
+The example on this page is using experimental mode in Docker 1.13 that can be enabled by using an experimental flag or option to Docker daemon, users can start the Docker daemon with or without experimental features.
+
+By default, experimental flag is turned off. To see the experimental flag, please check Docker version.
```

Comment:
By default, experimental mode is disabled. To see the current configuration, use the `docker version` command.